libweasyl/libweasyl/models/users.py

```diff
@@ -20,13 +20,21 @@
 log = logging.getLogger(__name__)
 
 
+_BLANK_AVATAR = "/static/images/avatar_default.jpg"
```

Comment:
This discussion goes beyond the scope of the code review but:

The existence of libweasyl has made some tooling easier to write (e.g. the thumbnail resize workers). It provides a distinction between the logic that forms a web-application and the logic that manipulates the data.

There is some cross over, but since it's usually in the direction of Weasyl doing something libweasyl should, rather than libweasyl doing something Web Application-like, it's generally remained clean.